# HG changeset patch
# User Fabian Groffen <grobian@gentoo.org>
# Date 1464576248 -3600
#      Mon May 30 03:44:08 2016 +0100
# Node ID 027213e185710690bb6f8a984990b694800689cd
# Parent  9f435837acfdb691b66cef01aaf484f66262b27c
[PATCH] xterm-set-only-window-title
fix xterm-titles patch to not set both window and icon title
---
 curs_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -r 9f435837acfd -r 027213e18571 PATCHES
--- a/PATCHES	Mon May 30 03:42:56 2016 +0100
+++ b/PATCHES	Mon May 30 03:44:08 2016 +0100
@@ -1,3 +1,4 @@
+patch-xterm-only-window-title-gentoo
 patch-lpr.collapse_flagged Lukas P. Ruf <lukas.ruf@lpr.ch>
 patch-dont-reveal-bcc-gentoo (modified version from Debian)
 patch-multiple-fcc-neomutt
diff -r 9f435837acfd -r 027213e18571 curs_main.c
--- a/curs_main.c	Mon May 30 03:42:56 2016 +0100
+++ b/curs_main.c	Mon May 30 03:44:08 2016 +0100
@@ -129,7 +129,7 @@ static const char *No_visible = N_("No v
    (option (OPTCOLLAPSEFLAGGED) || FLAGGED(header) == 0))
 
 /* de facto standard escapes for tsl/fsl */
-static char *tsl = "\033]0;";
+static char *tsl = "\033]2;";
 static char *fsl = "\007";
 
 /* terminal status capability check. terminfo must have been initialized. */
